compiler/hash-ir/src/ir.rs

```diff
@@ -836,11 +837,10 @@ pub struct Statement {
     /// The kind of [Statement] that it is.
     pub kind: StatementKind,
 
-    /// The [Span] of the statement, relative to the [Body]
-    /// `source-id`. This is mostly used for error reporting or
-    /// generating debug information at later stages of lowering
+    /// The location of the statement. This is mostly used for error reporting
+    /// or generating debug information at later stages of lowering
     /// beyond the IR.
-    pub span: Span,
+    pub span: AstNodeId,
```

Comment:
Hmm, should these kinds of fields now be called `origin`?

Reply:
yeah this is probably a better name for the fields.